packages/relay/src/lib/errors/JsonRpcError.ts

```diff
@@ -169,6 +169,11 @@ export const predefined = {
     code: -32601,
     message: 'Unsupported JSON-RPC method',
   }),
+  UNSUPPORTED_TRANSACTION_TYPE: new JsonRpcError({
+    name: 'Unsupported transaction type',
+    code: -32611,
```

Comment:
Q: How did we define the error code -32611 for this particular error?

Reply:
The codes after -32600 after are not part of any standard. I just picked the next unused one.